RecoTracker/LSTCore/interface/alpaka/Common.h

```diff
@@ -49,14 +49,26 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
       {0.4f, 0.4f, 0.4f, 0.4f, 0.4f, 0.4f, 0.4f, 0.4f, 0.18f, 0.18f, /*10*/ 0.18f, 0.18f, 0.18f, 0.18f, 0.18f},
       {0.4f, 0.4f, 0.4f, 0.4f, 0.4f, 0.4f, 0.4f, 0.4f, 0.4f, 0.18f, /*10*/ 0.18f, 0.18f, 0.18f, 0.18f, 0.18f}};
 
+  // Common constants used by both DNNs
+  HOST_DEVICE_CONSTANT float kEta_norm = 2.5f;
+  HOST_DEVICE_CONSTANT float kPhi_norm = kPi;
+  constexpr unsigned int kPtBins = 2;
+  constexpr unsigned int kEtaBins = 10;
```

Comment:
at least these two better be in some namespace (dnn) or just repeated in `t3dnn` and `t5dnn`: the choice of the number of bins seems mostly accidental for t3 and t5